start-mycroft.sh

```diff
@@ -180,80 +184,80 @@ function check-dependencies() {
 _opt=$1
 _force_restart=false
 shift
-if [[ "${1}" == "restart" ]] || [[ "${_opt}" == "restart" ]] ; then
+if [ "${1}" = "restart" ] || [ "${_opt}" = "restart" ] ; then
     _force_restart=true
-    if [[ "${_opt}" == "restart" ]] ; then
+    if [ "${_opt}" = "restart" ] ; then
         # Support "start-mycroft.sh restart all" as well as "start-mycroft.sh all restart"
         _opt=$1
     fi
     shift
 fi
-_params=$@
+_params=$*
 
-check-dependencies
+check_dependencies
 
 case ${_opt} in
     "all")
-        launch-all
+        launch_all
         ;;
 
     "bus")
-        launch-background ${_opt}
+        launch_background "${_opt}"
         ;;
     "audio")
-        launch-background ${_opt}
+        launch_background "${_opt}"
         ;;
     "skills")
-        launch-background ${_opt}
+        launch_background "${_opt}"
         ;;
     "voice")
-        launch-background ${_opt}
+        launch_background "${_opt}"
         ;;
 
     "debug")
-        launch-all
-        launch-process cli
+        launch_all
+        launch_process cli
         ;;
 
     "cli")
-        require-process bus
-        require-process skills
-        launch-process ${_opt}
+        require_process bus
+        require_process skills
+        launch_process "${_opt}"
         ;;
 
     # TODO: Restore support for Wifi Setup on a Picroft, etc.
     # "wifi")
-    #    launch-background ${_opt}
+    #    launch_background ${_opt}
     #    ;;
     "unittest")
-        source-venv
+        source_venv
         pytest test/unittests/ --cov=mycroft "$@"
         ;;
     "singleunittest")
-        source-venv
+        source_venv
         pytest "$@"
         ;;
     "skillstest")
-        source-venv
+        source_venv
         pytest test/integrationtests/skills/discover_tests.py "$@"
         ;;
     "vktest")
-        source "$DIR/bin/mycroft-skill-testrunner" vktest "$@"
+        bash "$DIR/bin/mycroft-skill-testrunner" vktest "$@"
```

Comment:
shouldn't this script define its own shebang? (ideally we wouldn't force `bash` from here)

Reply:
It does have it's own shebang so bash could be omitted... @krisgesling do you remember if there was any thought behind sourcing it? (Making it the same level as the rest of the function calls?)

Reply:
May have been related to something else I was trying and just got left in there. I can't think of any reason that it would be needed. Remove it I think.